File: C/Iniciante/1134 - Tipo de Combustível..c
/*
* Autor: Pierre Vieira
* Data de submissão: 10/02/2019 23:47:04
*/
#include<stdio.h>
int main(){
    int alcool, gasolina, diesel, opcao;
    alcool = gasolina =  diesel = opcao = 0;
    do{
        do{
            scanf("%d", &opcao);
        }while((opcao < 1) || (opcao > 4));
        if (opcao == 4){
            printf("MUITO OBRIGADO\n");
            printf("Alcool: %d\n",alcool);
            printf("Gasolina: %d\n",gasolina);
            printf("Diesel: %d\n",diesel);
        }
        else{
            if (opcao == 1){
                alcool++;
            }
            else if(opcao == 2){
                gasolina++;
            }
            else{
                diesel++;
            }
        }
    }while(opcao != 4);
}

File: C#/Iniciante/1134 - Tipo de Combustível.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 16/05/2020 21:37:57
*/
using System;

namespace URI {
    class Program {
        static void Main(string[] args) {
            string[] combustiveis = { "Alcool", "Gasolina", "Diesel" };
            int[] n_comb = {0, 0, 0};
            int n;
            Console.WriteLine("MUITO OBRIGADO");
            while (true) {
                n = int.Parse(Console.ReadLine());
                if (n <= 4 && n >= 1) {
                    if (n == 4) {
                        for (int i = 0; i < 3; i++) {
                            Console.WriteLine($"{combustiveis[i]}: {n_comb[i]}");
                        }
                        break;
                    }
                    n_comb[n - 1]++;
                }
            }
        }
    }
}
